Src/BootCamp.Chapter1/Program.cs

```diff
@@ -1,9 +1,48 @@
-﻿namespace BootCamp.Chapter1
+﻿using System;
+
+namespace BootCamp.Chapter1
 {
     class Program
     {
         static void Main(string[] args)
         {
+            var dataset = new[] { 999, 9, 99, 55, 1, 1337};
+            //var dataset = new int[1];
+            PrintDatasetResult("Start result: ", dataset);
+
+            //ArrayOperations.Sort(dataset);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5011611)